tools/codeowners-utils/Azure.Sdk.Tools.CodeownersUtils/Constants/OrgConstants.cs

```diff
@@ -0,0 +1,18 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Azure.Sdk.Tools.CodeownersUtils.Constants
+{
+    public class OrgConstants
+    {
+        // Azure is both the Org and the owner of the various repositories
+        public const string Azure = "Azure";
+        // Everyone with write permission in the Azure org either needs to be a direct user
+        // in the azure-sdk-write team or a user in a child team of azure-sdk-write. 
+        public const string AzureSdkWriteTeam = "azure-sdk-write";
+        public const string AzureOrgTeamConstant = $"{Azure}/";
```

Comment:
```suggestion
        public const string AzureOrgTeamConstant = "Azure/";
```

Any reason this isn't just a string contant like that?

Reply:
Other than if the org changes we'd only have to change the org constant. Six of one half a dozen of the other, I can hard code it.

Reply:
Oh I didn't realize that was pulling from the other constant named Azure, lol. In that case I'm fine with what you have.